### ISSUE TITLE
shutdownNow not allowed on managed exector service instances

### ISSUE THREAD

**Issue by arnovh** [eclipse-ee4j/jakartaee-tutorial-examples#17]
https://github.com/eclipse-ee4j/jakartaee-tutorial-examples/blob/9814044413785e7b3428fd0f0259a3f7c778ef69/concurrency/taskcreator/src/main/java/jakarta/tutorial/taskcreator/TaskEJB.java#L69

According to paragraph "3.1.6.1 Java EE Product Provider Requirements" of Concurrency Utiliries for Java EE JSR spec it is not allowed to call shutdownNow on managed exector service instances. Container should throw an IllegalStateException. I checked the logs of glassfish and didn't see such an exception however I am studying for Java EE Application Developer exam and seeing this code in the examples is confusing.

**Comment by bshannon** [comment]
Yes, this looks like a bug to me.  The two shutdownNow calls should be removed,
and we should verify that GlassFish is throwing the right exception (which is probably
being ignored in this @PreDestroy method).

@smillidge, as Project Lead for the Concurrency project, can you verify that this is
indeed a bug in the example?  If so, I'll update the example.  Thanks.
